packages/flutter/lib/src/widgets/animated_child_switcher.dart

```diff
@@ -53,17 +148,69 @@ class AnimatedChildSwitcher extends StatefulWidget {
   /// [duration].
   final Widget child;
 
+  /// The duration over which to perform the cross fade using [FadeTransition].
+  final Duration duration;
+
   /// The animation curve to use when fading in the current widget.
   final Curve switchInCurve;
 
   /// The animation curve to use when fading out the previous widgets.
   final Curve switchOutCurve;
 
-  /// The duration over which to perform the cross fade using [FadeTransition].
-  final Duration duration;
+  /// The builder function used to generate custom transitions for
```

Comment:
It would be more helpful to say what this builder function is for: A function that wraps the new [child] with an animation that transitions the [child] _in_ when the animation runs in the forward direction and _out_ when the animation runs in the reverse direction.

Reply:
Done.